Source/Core/TerrainTileEdgeDetails.js

```diff
@@ -0,0 +1,17 @@
+define([], function() {
+    'use strict';
+
+    function TerrainTileEdgeDetails() {
```

Comment:
Just style here and throughout, but I think we explicitly mark these as `@private` enough though it isn't required.

E.g. https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Cesium3DTilesetStatistics.js